components/shared/CurrentTime.tsx

```diff
@@ -0,0 +1,34 @@
+"use client";
```

Comment:
- Can you explain the purpose of this component ?

Reply:
I dont think displaying time is that necessary

Reply:
that is a good point, i will talk with eyosi on changing it